test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala

```diff
@@ -66,6 +67,7 @@ class CheckAllAnswersAndSubmitControllerSpec
     )
 
   lazy val controller = instanceOf[CheckAllAnswersAndSubmitController]
+  lazy val conf = instanceOf[ViewConfig]
```

Comment:
you don't need this `val` - you should just be able to use `viewConfig` where you need it